```
cryptcommon/ZrtpRandom.h:77:61: error: unknown type name 'bool'
   77 | int zrtp_AddEntropy(const uint8_t *buffer, uint32_t length, bool isLocked);
      |                                                             ^~~~
cryptcommon/ZrtpRandom.h:33:1: note: 'bool' is defined in header '<stdbool.h>'; this is probably fixable by adding '#include <stdbool.h>'
   32 | #include <sys/types.h>
  +++ |+#include <stdbool.h>
   33 | 
```

diff --git a/cryptcommon/ZrtpRandom.h b/cryptcommon/ZrtpRandom.h
index ee8640a..44d0b6b 100644
--- a/cryptcommon/ZrtpRandom.h
+++ b/cryptcommon/ZrtpRandom.h
@@ -30,6 +30,7 @@
 #endif
 #include <stdint.h>
 #include <sys/types.h>
+#include <stdbool.h>
 
 #ifdef __cplusplus
 class ZrtpRandom {
@@ -82,4 +83,4 @@ int zrtp_getRandomData(uint8_t *buffer, uint32_t length);
 }
 #endif
 
-#endif /* ZRTPRANDOM */
\ No newline at end of file
+#endif /* ZRTPRANDOM */
